manifests/c/CPUID/CPU-Z/2.05/CPUID.CPU-Z.installer.yaml

```diff
@@ -0,0 +1,77 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: CPUID.CPU-Z
+PackageVersion: 2.05
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-03-01

```

Comment:
```suggestion
```
My apologies, I did not see the ReleaseDate was already present at the end of file